src/components/CJobs.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <div class="px-4 mt-10 pt-2">
+    <table class="w-full">
+      <thead>
+        <tr class="bg-gray-100 border-4 h-8">
+          <th v-for="column in columns" :key="column.name" class="border-4 px-4 py-2">
+            <div class="ml-2 text-sm  font-semibold  leading-5">
+              {{ column.label }}
+            </div>
+          </th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr v-for="(job, index) in jobs" :key="index">
+          <td class="border px-4 py-2" v-for="column in columns" :key="column.name">
+            <pre class="text-xs whitespace-normal ml-2 bg-white"> {{ job[column.name] }}</pre>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+  </div>
+</template>
+<script>
+export default {
+  name: 'CJobs',
+  data() {
+    return {
+      columns: [
+        { label: 'Actor Name', name: 'actor_name' },
```

Comment:
wiating for the test script for better testing but it seems to me this data could be formatted.